### DISCUSSION TITLE
How does Album search supposed to work?  I seem to be getting mixed results.

### DISCUSSION THREAD

**Discussion by hiro5id** [photoprism/photoprism#2004] [Q&A] [answered]
When I search for `albums:"Julian - Baby"` I get results from 2 albums:

from album name `Julian - Baby`
and from
album name `Baby Photos Sebestyen Family`

Is it because the word "baby" shows up in both?  How can I tell it to only find photos from a specific album.  The reason I want to do this is because I want to combine it with other filters.  (Since I am unable to run queries when I click into the album itself)

**Comment by lastzero** [comment, marked as answer]
It's currently a fuzzy search. We could change the behavior if everyone is happy with that.

**Reply by hiro5id** [reply]
I for one vote "yes please" :)

**Reply by hiro5id** [reply]
Perhaps if people need a fuzy search like feature for album names, perhaps it could work like this:
* searching with `|` or `&` to mach on words just like keyword search does like this `albums:"baby&car&seat"`
* but searching in quotes `albums:"album name"` would find exact album name (preferably case insensitive if possible)

**Reply by hiro5id** [reply]
Or maybe leave the current behaviour alone, and introduce a new search tag like `albumfullname:"the album name"` which would match the whole album name.

**Reply by graciousgrey** [reply]
As a workaround you can search for the album uid (you find it in the url of an album), e.g. for this album https://demo.photoprism.app/albums/aqoweuibbs3lhs40/birds you can use album:aqoweuibbs3lhs40

**Comment by Jayknightfr** [comment]
I'd love to have some advanced search filters so i can combine search like "face/place/year" etc, or just by location, ie get all my pictures taken in China.

The "colors", "cards" and "categories" filters are kinka odd, ngl.

**Reply by graciousgrey** [reply]
That's already possible using our search filters (https://docs.photoprism.app/user-guide/organize/search/#search-filter-overview) `country:cn person:"Jane Doe" year:2022`

**Reply by Jayknightfr** [reply]
Thanks! I didn't know about that. 
But couldn't it be possible to have most of them as filters in the UI?

**Reply by graciousgrey** [reply]
That's planned for the future. We can't just add more of the filters we currently use as this would use too much space especially on mobile devices. We envision something where users get suggestions for filters when clicking into the search field.
